oci.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import logging
+
+from oci import config
+from oci import gerrit
+from oci import jenkins
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='A command line tool for working with Ovirt CI')
+
+    parser.add_argument(
+        '--verbose',
+        help="increase output verbosity",
+        action="store_true")
+    subparsers = parser.add_subparsers(title="commands")
+
+    build_artifacts = subparsers.add_parser(
+        "build-artifacts",
+        help="start the build-artifacts stage")
```

Comment:
This actually start build artifacts and wait until it completes. But I think the help can be something like "build artifacts for a change"